### PR TITLE
Moved JellyfishCreditMemo package from FOS to FOO

### PR DESCRIPTION
**Pull request by julianzimmermann**

### CODE REVIEW

File: bundles/jellyfish-credit-memo/composer.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      "name": "József Geng",
+      "email": "gengjozsef86@gmail.com"
+    },
+    {
+      "name": "Julian Zimmermann",
+      "email": "dev@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.3"
```
Comment (daniel-rose) [inline bundles/jellyfish-credit-memo/composer.json:16]: Have this module no dependencies?

File: bundles/jellyfish-credit-memo/src/FondOfOryx/Shared/JellyfishCreditMemo/JellyfishCreditMemoConstants.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+interface JellyfishCreditMemoConstants
+{
+    public const SALES_ORDER_ITEM_STATE_REFUNDED = 'SALES_ORDER_ITEM_STATE_REFUNDED';
+    public const BASE_URI = 'JELLYFISH_CREDIT_MEMO:BASE_URI';
+    public const TIMEOUT = 'JELLYFISH_CREDIT_MEMO:TIMEOUT';
+    public const USERNAME = 'JELLYFISH_CREDIT_MEMO:USERNAME';
+    public const PASSWORD = 'JELLYFISH_CREDIT_MEMO:PASSWORD';
+    public const DRY_RUN = 'JELLYFISH_CREDIT_MEMO:DRY_RUN';
+    public const SYSTEM_CODE = 'JELLYFISH_CREDIT_MEMO:SYSTEM_CODE';
```
Comment (daniel-rose) [inline bundles/jellyfish-credit-memo/src/FondOfOryx/Shared/JellyfishCreditMemo/JellyfishCreditMemoConstants.php:13]: Good Work. 👍

File: bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/JellyfishCreditMemoFacadeInterface.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace FondOfOryx\Zed\JellyfishCreditMemo\Business;
+
+interface JellyfishCreditMemoFacadeInterface
+{
+    /**
+     * Export Credit Memos To Jellyfish
```
Comment (daniel-rose) [inline bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/JellyfishCreditMemoFacadeInterface.php:8]:
```php
Specifications:
- Exports credit memos to jellyfish

@api

```

File: bundles/jellyfish-credit-memo/composer.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "require-dev": {
```
Comment (sourcecube) [inline bundles/jellyfish-credit-memo/composer.json:18]: It seems odd that there are no dependencies listed here.. I can see at least two, is that correct?

File: bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/Model/Exporter/CreditMemoExporter.php [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+            foreach ($creditMemoCollectionTransfer->getCreditMemos() as $creditMemoTransfer) {
+                if ($this->isValidForExport($creditMemoTransfer) === false) {
+                    continue;
+                }
+
+                $jellyfishCreditMemo = $this->map($creditMemoTransfer);
+                $this->adapter->sendRequest($jellyfishCreditMemo);
+
+                $jellyfishCreditMemo->setExportState(static::CREDIT_MEMO_EXPORT_STATE_COMPLETE);
+                $this->jellyfishCreditMemoEntityManager->updateExportState($jellyfishCreditMemo);
```
Comment (daniel-rose) [inline bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/Model/Exporter/CreditMemoExporter.php:99]: Maybe you can use an protected method for this logic.

**Review by sourcecube — COMMENTED** [review]
Tests missing..

File: bundles/jellyfish-credit-memo/composer.json (hunk 1 above)
Comment (daniel-rose) [reply bundles/jellyfish-credit-memo/composer.json:16]: You use brigdes to util-encoding and sales.

**Review by daniel-rose — APPROVED** [review]
Do we neet a composer.phar file in our project?

File: bundles/jellyfish-credit-memo/.gitignore [hunk 6]
```diff
@@ -0,0 +1,6 @@
+.idea
```
Comment (sourcecube) [inline bundles/jellyfish-credit-memo/.gitignore:1]: File can be removed.. it will inherit from the parent.

**Pull request merged**